Yoast/wordpress-seo
# Improve copy Ryte health check

## Context


* To inform the users of the fact that the Ryte Indexability status can only be fetched every 15 seconds. Adjusts the copy and adds an alert box.

## Summary

This PR can be summarized in the following changelog entry:

* Improves the copy for the "cannot fetch" response of the Ryte health check.

## Relevant technical choices:

* Adjusts copy and adds alert box, as discussed with @JessieHenkes 
* Uses `Alert_Presenter` class (see #14803)
* Adjusts tests accordingly
* Copy that was decided on:
> Ryte offers a free indexability check for Yoast SEO users  and right now it has trouble determining whether search engines can find your site. This could have several (legitimate) reasons and is not a problem in itself. If this is a live site, it is recommended that you figure out why the Ryte check failed. 
>
> As the indexability status of your website can only be fetched from Ryte evert 15 seconds, a first step could be to wait at least 15 seconds and refresh the Site Health page. If that did not help, [read more about troubleshooting search engine visibility](url). 
>
> [Go to Ryte to analyze your entire site](url)
## Test instructions

This PR can be tested by following these steps:

1. In `class-ryte-request.php` >  add `$target_url = 'https://nonexisting.yoast.com/';` on the first line of the `get_remote()` function
2. In the backend go to `Tools` > `Site Health`
3. See the ryte test within the `recommended improvements` section: "An error occurred while checking whether your site can be found by search engines" <img width="637" alt="error500" src="https://user-images.githubusercontent.com/43582255/75255357-15e86a80-57e2-11ea-8ea3-c3e3204c6be6.png">
4. Refresh the `Site Health` page within 15 seconds and see the Ryte test response change to "Ryte cannot determine whether your site can be found by search engines", as shown in the image below <img width="602" alt="Screenshot 2020-04-23 at 14 48 03" src="https://user-images.githubusercontent.com/43582255/80106324-008d8380-857a-11ea-8ce2-df3b5845c74c.png">
5. Make sure both links work


## UI changes

## Documentation

## Quality assurance


Fixes #14430
